packages/core/src/atom.ts

```diff
@@ -671,7 +671,7 @@ function onCall(this: Action, cb: Fn): Unsubscribe {
   })
 }
 
-export function atom<T>(initState: (ctx: CtxSpy) => T, name?: string): Atom<T>
+export function atom<T>(callback: (ctx: CtxSpy) => T, name?: string): Atom<T>
```

Comment:
here are my options:
- reducer, because the second argument is the previous state
- computer, because when we pass a function we turn an atom into a computed atom

Reply:
@artalar

Reply:
`computed` is the right name